package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
Thanks for comments.
run-s is supposed to be used for sequential tasks. So if I replace npm-run-all with run-s, then it never reaches any task after running the first one, as they are all 'watch' tasks and don't finish. Probable you meant to use run-s to compose tasks in more complex way to get rid of multiple nodemon invocation. Then I will try to think about it.